src/config/chainConfig.ts

```diff
@@ -83,4 +83,16 @@ export const chainConfig: IChainDefinitions = {
   [CHAIN_ID.HARMONY]: {
     rpcUrl: process.env.HARMONY_RPC_URL || "https://api.harmony.one",
   },
+  [CHAIN_ID.HECO]: {
```

Comment:
if you can add some default rpcs please do
use chainlist, search with chain ids and pick any rpc:

https://chainlist.org/